core/src/test/java/org/elasticsearch/cluster/metadata/WildcardExpressionResolverTests.java

```diff
@@ -125,6 +126,47 @@ public void testAll() {
         assertThat(newHashSet(resolver.resolve(context, Arrays.asList("_all"))), equalTo(newHashSet("testXXX", "testXYY", "testYYY")));
     }
 
+    // issue #23960
```

Comment:
@jasontedor Good point! I just followed the lead of another test in this class which is abbreviated only with the issue number. 

May I assume that generally a test should be commented with a short description why the test has been introduced (and maybe the issue for further reference)?